Oops.  End of `gmake` invocation.

```
;; Wrote the memory image into ./bin/clisp/shen (8,856,736 bytes)
Bytes permanently allocated:            164,192
Bytes currently in use:               4,996,752
Bytes available until next GC:           77,296
Bye.
./bin/clisp/shen --clisp-m 10MB -e "(do (cd \"kernel/tests\") (load \"README.shen\") (load \"tests.shen\"))"
GNU CLISP: invalid argument: '-e'
GNU CLISP: use '-h' for help
gmake: *** [Makefile:150: test-clisp] Error 1
```

Don't know why it doesn't like `-e`.  Any ideas?

I looked at the Clisp --help and it doesn't list `-e`, but lists `-x` as the option to eval expressions.  But that doesn't seem to work either:

```
$ ./bin/clisp/shen --clisp-m 10MB -x "(do (cd \"kernel/tests\") (load \"README.shen\") (load \"tests.shen\"))" 
module 'syscalls' requires package OS.
```

This is on OpenBSD 6.2 amd64.  `gmake sbcl` and `gmake ecl` are both completing successfully though.

--- a/Makefile
+++ b/Makefile
@@ -45,7 +45,7 @@ else
 	ifeq ($(OSName),freebsd)
 		All=ccl ecl sbcl
 	else ifeq ($(OSName),openbsd)
-		All=clisp ecl sbcl
+		All=ecl sbcl
 	else ifeq ($(OSName),netbsd)
 		All=clisp ecl
 	endif

--- a/README.md
+++ b/README.md
@@ -20,7 +20,7 @@ Bug reports, fixes and enhancements are welcome. If you intend to port Shen to a
 
 You will need to have recent versions of the Common Lisp implementations you want to work with installed and available as the `Makefile` requires. Installation is different depending on operating system.
 
-Support for Common Lisp implementations varies over different operating systems. By default, only implementations available in binary form are built with the `make all` command. Check the value of the `All` variable in the `Makefile` by running `make env` to which are supported on your machine.
+Support for Common Lisp implementations varies over different operating systems. By default, only implementations available in binary form are built with the `make all` command. Check the value of the `All` variable in the `Makefile` to see which are supported for your OS and architecture.
 
 Check the project page for any CL implementation to build from source if necessary.
 

